app/src/components/billing/BillingContentTabs/Invoices/Invoices.tsx

```diff
@@ -0,0 +1,50 @@
+import { Card, VStack, Text, Icon, Heading } from "@chakra-ui/react";
+import { useInvoices } from "~/utils/hooks";
+import InvoicesTable from "./InvoicesTable";
+import NewTestInvoiceButton from "./NewTestInvoiceButton";
+import { FaTable } from "react-icons/fa";
+
+const Invoices = () => {
+  const query = useInvoices();
```

Comment:
Nit: since you're only using data from the query, I would rewrite this as something like:

```
invoices = useInvoices().data;
```

Reply:
I use it to call `query.isLoading` to show "No invoices yet" only if the page is loaded.